index.html

```diff
@@ -1,657 +1,1131 @@
-                            </svg>
+                      </div>
+
+                      <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <a id="button-61928ed28f" class="cmp-button" href="/content/leaderimpact/lang/en2/join-a-group.html">
+                          <span class="cmp-button__text">Join a Group</span>
                         </a>
-                        <a href="" data-cmp-clickable="" class="social-link-item">
-                            <svg width="23" height="23" viewBox="0 0 23 23" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path d="M3.04167 0.625C1.70646 0.625 0.625 1.70646 0.625 3.04167V19.9583C0.625 21.2935 1.70646 22.375 3.04167 22.375H19.9583C21.2935 22.375 22.375 21.2935 22.375 19.9583V3.04167C22.375 1.70646 21.2935 0.625 19.9583 0.625H3.04167ZM3.04167 3.04167H19.9583V19.9583H3.04167V3.04167ZM6.39998 4.63232C5.36444 4.63232 4.74325 5.25482 4.74325 6.08374C4.74325 6.91266 5.36452 7.5328 6.29614 7.5328C7.33168 7.5328 7.95288 6.91266 7.95288 6.08374C7.95288 5.25482 7.33161 4.63232 6.39998 4.63232ZM4.82585 9.08333V17.5417H7.875V9.08333H4.82585ZM10.3908 9.08333V17.5417H13.4399V12.9184C13.4399 11.5421 14.4219 11.3442 14.7167 11.3442C15.0115 11.3442 15.8 11.64 15.8 12.9184V17.5417H18.75V12.9184C18.75 10.2637 17.5703 9.08333 16.095 9.08333C14.6196 9.08333 13.8327 9.5746 13.4399 10.2633V9.08333H10.3908Z" fill="#9C9FA1"/>
-                            </svg>
+                      </div>
+
+                      <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <a id="button-9752993433" class="cmp-button" href="/content/leaderimpact/lang/en2/start-a-group.html">
+                          <span class="cmp-button__text">Start a Group</span>
                         </a>
+                      </div>
+                      <!-- FOOTER NAVIGATION WRAP -->
+                      <div class="container responsivegrid navigation-wrap aem-GridColumn aem-GridColumn--default--6 layout-grid">

```

Comment:
@Prystal254 I'm not sure why this is in the same container as the footer buttons.